lib/rest-client/all/rest-client.rbi

```diff
@@ -0,0 +1,3 @@
+class RestClient::Request
+    def self.execute(args, &blk); end
```

Comment:
Could you add a type signature here?

Reply:
I am just starting to get sorbet into a big codebase. I can get back to that once I have decent types elsewhere.